scripts/tests/harness.sh

```diff
@@ -62,6 +62,18 @@ function forest_run_node_stateless_detached {
   $FOREST_PATH --detach --chain calibnet --encrypt-keystore false --config "$CONFIG_PATH" --log-dir "$LOG_DIRECTORY" --save-token ./stateless_admin_token --skip-load-actors --stateless
 }
 
+function forest_run_node_mapping_ttl_detached {
```

Comment:
Are you sure it belongs to the harness? That is to say, do you think it will be used anywhere outside of `calibnet_eth_mapping_check.sh` in the future?

Reply:
Not really, will move it.